src/components/FeatureCategories.js

```diff
@@ -1,38 +1,41 @@
 import { Link } from "react-router-dom";
-import { useFilter } from "../contexts/FilterContext";
-import { ProductCategoryData } from "../data";
+import { useFilter } from "contexts/FilterContext";
+import { useCategories } from "contexts/CategoriesContext";
 
 export const FeatureCategories = () => {
   const { dispatchFilter } = useFilter();
+  const { categories } = useCategories();
   return (
     <section className="section centered-text grid-ctr w-95p m-t-3 m-b-5">
       <h2 className="title colored-text h1 m-v-1">
         <span className="circle"></span>Because we <span className="complementary-color-text">Care</span>
       </h2>
       <div className="product-container grid-3-col full-wd m-v-2 m-b-5">
-        {ProductCategoryData?.map(({ id, name, description, image, link }) => {
+        {categories?.map(({ _id, tag, categoryName, description, contentColor, image, link }) => {
           return (
             <Link
               to={link}
-              key={id}
+              key={_id}
               href="pages/products.html"
```

Comment:
You are using  to and href both in Link tag. I think href should not be there. In browser also when clicking on link your app is routing to html page not the route that you have defined in to={link} tag

Reply:
okay yes missed this. I've removed the `href` from here.